azure-pipelines.yaml

```diff
@@ -15,32 +15,21 @@ parameters:
   - name: fortifyScan
     displayName: Run Fortify scan
     type: boolean
-    default: true
-  - name: fortifyQualityGateLevel
-    displayName: Fortify quality gate level (0 for non-blocking)
-    type: number
-    default: 1
+    default: false
     
 variables:
   applicationName: 'mi-core-lib'
   azureSubscriptionPrefix: 'DTS-SHAREDSERVICES'
   ${{ if not(eq(variables['Build.SourceBranch'], 'refs/heads/master')) }}:
     environment: ${{ parameters.environment }}
-    fortifyScan: ${{ parameters.fortifyScan }}
-    fortifyQualityGateLevel: ${{ parameters.fortifyQualityGateLevel }}
   ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
     environment: 'stg'
-    fortifyScan: true
-    fortifyQualityGateLevel: ${{ parameters.fortifyQualityGateLevel }}
     
 stages:
-  - ${{ if parameters.fortifyScan }}:
-    - template: ./pipeline-steps/templates/run-fortify-scan.yaml
-      parameters:
-        azureSubscriptionEndpoint: "${{ variables.azureSubscriptionPrefix }}-${{ upper(variables.environment) }}"
-        azureVault: "mi-vault-${{ lower(variables.environment) }}"
-        vaultAgentPool: "hmcts-ss-${{ lower(variables.environment) }}"
-        applicationName: "${{ variables.applicationName }}"
-        qualityGateLevel: ${{ variables.fortifyQualityGateLevel }}
-        technologyStack: 'JS/TS/HTML'
-        technologyVersion: ''
+  - template: ./pipeline-steps/templates/run-fortify-scan.yaml
```

Comment:
Conditions for fortify scan is now enforced by the fortify job on PR and master builds. Now fortify scans won't run unless the flag is manually set to true instead of the default false, or its a PR / master run.

Reply:
Will make changes for all repos similar to this so the condition is now controlled by the core-lib